the-collab-lab/tcl-28-smart-shopping-list
# Ks empty list

## Description


- Created full shopping basket image to match our empty basket from Figma
- Empty basket displays when shopping list is empty
- Full basket displays when shopping list contains items
- Removed unused images from `public/img` folder

## Related Issue


closes #49 

## Acceptance Criteria

- [x] Add style to list view when there are no items in the list

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |

## Updates

### Before


![Screen Shot 2021-09-01 at 7 32 20 AM](https://user-images.githubusercontent.com/40218057/131680520-074eb829-eeb9-491a-bd5e-f73fe948d3a1.png)

![Screen Shot 2021-09-01 at 7 31 42 AM](https://user-images.githubusercontent.com/40218057/131680386-d299c964-881c-4fef-907e-8def23b77659.png)

### After


![Screen Shot 2021-09-01 at 7 28 59 AM](https://user-images.githubusercontent.com/40218057/131679986-ac73bbc3-a811-447e-96c5-5949ae9fef43.png)

![Screen Shot 2021-09-01 at 7 30 26 AM](https://user-images.githubusercontent.com/40218057/131680201-c67c9ce2-1d9a-4ad1-90a4-2ebb5119f07f.png)

## Testing Steps / QA Criteria


- checkout `ks-empty-list` branch or visit preview url
- create a list with no items, you should see an empty basket in the list view
- add items to your list, you should see a full basket in the list view
